README.md

```diff
@@ -83,6 +84,21 @@ export default ProductPrice
 
 Also, all dependencies needed should be inserted inside the react/package.json
 
+## Styles API
+This app has CSS customization through `CSS Modules`. CSS Modules is a CSS file in which all class names and animation names are scoped locally by default. You can read more about CSS Modules [here](https://github.com/css-modules/css-modules).
+
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and major version. Like `productNameContainer` token declared in `ProductName`, generate the classname `vtex.store-components-3-x-productNameContainer`.
+
```

Comment:
Add a text explaining that each Component has its own README with the tokens described.